expertise/service/or_queue.py

```diff
@@ -0,0 +1,164 @@
+import os, openreview, shutil
```

Comment:
do you still need this file?

Reply:
Oh right I meant to remove the old queues - I can do that right now